crypto/evp_extra/p_pqdsa_asn1.c

```diff
@@ -235,7 +262,7 @@ const EVP_PKEY_ASN1_METHOD pqdsa_asn1_meth = {
   pqdsa_pub_cmp,
   pqdsa_priv_decode,
   pqdsa_priv_encode,
-  NULL /*priv_encode_v2*/,
+  pqdsa_priv_encode_seed,
```

Comment:
But this is PKCS#8 v1 encoding you are doing here, so this shouldn't be set as the v2 encoding function?